public/web.config

```diff
@@ -23,5 +23,12 @@
         <remove fileExtension=".woff2" />
         <mimeMap fileExtension=".woff2" mimeType="application/x-font-woff2" />
     </staticContent>
+        <directoryBrowse enabled="true" />
```

Comment:
This probably isn't part of this change. Happy to take it as a separate PR if it's something we need though.

Reply:
yup sorry this file has been included by mistake. will revert it  to the original one